### ISSUE TITLE
Cancelling a running aggregate graph in console CLI crashes the console

### ISSUE THREAD

**Issue by tznind** [HicServices/RDMP#374]
Steps to repro:
1. Create a complicated aggregate and start it running
2. Cancel running graph or just tab around the UI

Error was:

```
The given key '0' was not present in the dictionary.                        
   at System.Collections.Generic.SortedList`2.get_Item(TKey key)            
   at Terminal.Gui.MainLoop.RunTimers()                                     
   at Terminal.Gui.MainLoop.MainIteration()                                 
   at Terminal.Gui.Application.RunLoop(RunState state, Boolean wait)        
   at Terminal.Gui.Application.Run(Toplevel view, Func`2 errorHandler)      
   at Rdmp.Core.CommandLine.Gui.ConsoleGuiActivator.ShowGraph(AggregateConfiguration aggregate)                                                         
   at Rdmp.Core.CommandExecution.AtomicCommands.ExecuteCommandExecuteAggregateGraph.Execute()                                                           
   at Rdmp.Core.CommandLine.Gui.ConsoleMainWindow.<>c__DisplayClass23_0.<Menu>b__3()
```

This is then followed up by:

```
An item with the same key has already been added. Key: 637571874286529981 (Parameter 'key')
```

Offending code might be:
```csharp

var timeout = Application.MainLoop.AddTimeout(TimeSpan.FromSeconds(1),(s)=>
{
    this.SetNeedsDisplay();
    return true;
});
```

**Comment by tznind** [comment]
This issue may be fixed in 1.1.1 (See: https://github.com/migueldeicaza/gui.cs/issues/1307)

**Comment by tznind** [comment]
`ConsoleGuiViewGraph` is a subclass of `ConsoleGuiSqlEditor`

Task cancellation/crash handling now happens in a ContinueWith loop so should be invoked on the correct Thread:

https://github.com/tznind/RDMP/blob/03c2d46d33b420b796d9a846bf5cc1d519aac274/Tools/rdmp/CommandLine/Gui/ConsoleGuiSqlEditor.cs#L264-L269

Does not seem to still be an issue.
